Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM mcr.microsoft.com/dotnet/core/runtime:3.1 AS base
+WORKDIR /app
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS build
+WORKDIR /src
+COPY ["TheoremSlackBot.csproj", "./"]
+RUN dotnet restore "./TheoremSlackBot.csproj"
+COPY . .
+WORKDIR "/src/."
+RUN dotnet build "TheoremSlackBot.csproj" -c Release -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "TheoremSlackBot.csproj" -c Release -o /app/publish -p:PublishReadyToRun=false
```

Comment:
Why readytorun=false?

Reply:
The R2R binary caused an error in the dotnet sdk container for some reason. Disabling it instead of trying to figure out why it was happening was a quick way to unblock myself 🤷